aequilibrae/paths/linear_approximation.py

```diff
@@ -10,16 +10,14 @@
 from aequilibrae.paths.AoN import triple_linear_combination, triple_linear_combination_skims
 from aequilibrae.paths.AoN import copy_one_dimension, copy_two_dimensions, copy_three_dimensions
 from aequilibrae import logger
-
-try:
+import scipy
+if int(scipy.__version__.split('.')[1]) >= 3:
```

Comment:
You could also consider using ```semver``` to avoid edge cases

![image](https://user-images.githubusercontent.com/151124/75495123-891ff580-5a09-11ea-92d2-c2af31822929.png)

This code would flag scipy version 2.1 as old - ```if semver.compare(semver.parse_version_info(scipy.__version__), "1.4.0") < 0```

Reply:
I looked into similar stuff, but there is nothing like that available in the Python distribution that comes with QGIS for Windows, so I am a bit stuck.  I actually hope to deprecate the use of SciPy older than 1.3 as soon as the QGIS Windows distribution gets updated.